packages/client/src/views/SysAdmin/Performance/WorkflowStatus.tsx

```diff
@@ -82,6 +82,9 @@ const INITIAL_SORT_MAP = {
 
 const DEFAULT_APPLICATION_STATUS_PAGE_SIZE = 25
 
+if (!window.config.EXTERNAL_VALIDATION_WORKQUEUE) {
+  delete StatusMapping['WAITING_VALIDATION']
```

Comment:
Same comment as above, let's filter it out instead of removing it from the mapping object